### PR TITLE
Lb/add specs for error pages

### PR DESCRIPTION
**Pull request by elceebee**
## Context

Improving test coverage throughout the application

## Changes proposed in this pull request

- Added the too_many_requests template (doh, we had a route, but not template!)
- Added tests for the error pages, for both services

## Guidance to review

Tests pass
If you test the routes locally for either service, you should see the correct error pages. 

## Link to Trello card

https://trello.com/c/p3g63rPY

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by elceebee** [comment]
[[Cov] Add specs for the error pages](https://trello.com/c/p3g63rPY/189-cov-add-specs-for-the-error-pages)

File: spec/system/error_pages_spec.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    expect(page).to have_content "Try again later."
+    expect(page).to have_content "If you reached this page after submitting information then it has not been saved. You’ll need to enter it again when the service is available."
+    expect(page.driver.status_code).to eq 500
+  end
+
+  def then_i_see_429_content
+    expect(page).to have_title "Sorry, there’s a problem with the service"
+    expect(page).to have_content "Sorry, there’s a problem with the service"
+    expect(page).to have_content "Try again later."
+    expect(page).to have_content "If you have any questions, please email us at becomingateacher@digital.education.gov.uk"
```
Comment (Nitemaeric) [inline spec/system/error_pages_spec.rb:83]: Should this also have status code of 429?

Comment (Nitemaeric) [reply spec/system/error_pages_spec.rb:83]: Not enough to be a blocker for me though.

**Review by Nitemaeric — APPROVED** [review]
🙌

**Pull request merged**